README.md

```diff
@@ -1 +1,28 @@
-# funding-service-design-post-award-docker-runner
\ No newline at end of file
+# funding-service-design-post-award-docker-runner
+
+## Prerequisites
+*  [Docker Desktop](https://www.docker.com/products/docker-desktop/) installed and running
+*  All funding-service-design apps (listed as `context` keys in [docker-compose.yml](docker-compose.yml) must be checked out in the parent directory on this repository
+
+## How to run
+* Copy `.env.example` to `.env` and ask another team member for the missing secret value
```

Comment:
does this example file exist? I can't see it

Reply:
Think maybe we should remove this reference until we have some secrets?

Reply:
can do although I personally think we can just leave it because I'm using `.env` already in my branch for authentication so it'll be added very soon I expect